pkg/network/network.go

```diff
@@ -122,6 +122,14 @@ const (
 	// QueueProxyUserAgent is the user-agent header value set in probe requests sent
 	// from queue-proxy.
 	QueueProxyUserAgent = "Knative-Queue-Proxy-Probe"
+
+	// IngressReadinessUserAgent is the user-agent header value
+	// set in probe requests for Ingress status.
+	IngressReadinessUserAgent = "Knative-Ingress-Probe"
+
+	// AutoscalingUserAgent is the user-agent header value set in probe
+	// requests sent by autoscaling implementations.
+	AutoscalingUserAgent = "Knative-Autoscaling-Probe"
```

Comment:
does this constant need to live here, rather than in the `scaler.go` ?

Reply:
It can like closer to the usage if you prefer. I put it here because that seemed like the existing practice.

Reply:
Mostly feels like we don't need constants at all, but I don't have a strong opinion.